s3_sync/s3_sync.py

```diff
@@ -0,0 +1,179 @@
+#!/usr/bin/env python
+
+"""
+
+Read in setting values from ini file and then run s3 sync between local folder
+and specified S3 bucket. Send results to local logfile & notify slack channel.
+
+Use:
+python s3_sync.py -i <settings_file.ini>
+
+"""
+try:
+    import argparse
+    import json
+    import logging
+    import os
+    import requests
+    import subprocess
+    import sys
+    from configparser import ConfigParser, ExtendedInterpolation
+    from logbook import Logger, RotatingFileHandler
+except ImportError as err:
+    sys.exit(err)
+
+# Instantiate argparse to get settings_file as argument
+parser = argparse.ArgumentParser(description='.')
+parser.add_argument('-i', dest="settings_file", required=True,
+                    help='ini file containing configs', metavar='FILE')
+args = parser.parse_args()
+settings_file = args.settings_file
+
+# Read settings_file
+config = ConfigParser(interpolation = ExtendedInterpolation())
+config.read(settings_file)
```

Comment:
If it doesn't exist, it would fail to read. Don't currently have any checks on whether it's malformed